packages/atomic/src/global/global.pcss

```diff
@@ -22,4 +22,8 @@
   .with-parentheses::after {
     content: ')';
   }
+
+  .link {
+    @apply text-secondary hover:text-secondary-light hover:underline focus:text-secondary-light focus:underline;
```

Comment:
It might be worth checking out with the accessibility theme that, on Windows with high contrast mode on, the hover effect is still visible.

However, we should probably eventually do that for the whole page anyways.

Reply:
Accessibility recommends not only using color, we have an underlined value on focus/hover